src/domain/Subscription.ts

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
The [copyright header](https://github.com/ideal-lab5/idn-sdk/blob/a994a9bd2c0394c811256e5177d75686226d7a3e/client/consensus/randomness-beacon/src/gossipsub.rs#L1) is missing.